applications/nrf_desktop/src/modules/dfu.c

```diff
@@ -31,7 +31,9 @@ LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_CONFIG_CHANNEL_DFU_LOG_LEVEL);
 #endif
 
 
-#define FLASH_PAGE_SIZE 0x1000
+#define FLASH_PAGE_SIZE_LOG2	12
```

Comment:
this is magic value with magic name

Reply:
but i know what is doing as know the flash layout

Reply:
I do not know what is it doing because I don't know flash layout... so please provide somethin more meaningful

Reply:
Will adding a comment that `FLASH page size is equal 2^12` help?
Before erasing i may also add comment that we need to erase whole FLASH pages.

Reply:
due to fact that similar names are use also in Zephyrs code - let it stay as it